theme/utils/neon2wp.php

```diff
@@ -35,7 +35,9 @@ function transformFields($fields, $prefix = NULL) {
 	}
 
 	$str = file_get_contents($path);
-	$str = Nette\Utils\Strings::replace($str, '~%post_id%~', isset($_GET['post']) ? $_GET['post'] : NULL);
+	if(isset($_GET['post']) && is_numeric($_GET['post'])) {
```

Comment:
Počkej, on někde wordpress zapisuje do `$_GET`? Protože za normálních okolností Ti z `$_GET` vždycky vyleze string, ne? A ano, samozřejmě by se to stát nemělo, ale přesně kvůli tomuhle tam ta kontrola je… 😃